design/Frontend-Design.md

```diff
@@ -1 +1,46 @@
-# TBD
\ No newline at end of file
+### Clean code
+
+#### Variables
+Give meaningful names to variables
+
+#### Functions
+1. Function name should describe what it does.
+2. A function should do only one thing.
+3. Limit the amount of function parameters, preferably two. if you need more you can send the parameters inside an object.
+4. If you find yourself writing the same code more than once, even if its a little bit different, try to generalize it in to a function.
+5. Functions should be pure
+
+#### Classes
+
+#### React components
+1. Make your React component as short as possible
+2. Name Your Components Using Standard Naming Conventions
+3. Reduce the number of props to the minimum
```

Comment:
[Sand]
Should we talk about prop-types here or that's not a part of clean code?